operator/src/index.js

```diff
@@ -7,8 +7,31 @@ const {Store} = require("./store");
 
 const port = process.env.WS_PORT;
 
-async function startServer(provider, enigmaUrl, contractAddr, scAddr, threshold, accountIndex = 0) {
-    const api = new OperatorApi(provider, enigmaUrl, contractAddr, scAddr, threshold, accountIndex);
+// This method tries to create an ethereum account from a private key provided from the environment.
+// If no private key is found, it attempts to fetch the unlocked accounts from the ethereum node,
+// and chooses the one whose index has been passed in the constructor as accountIndex.
+// If that fails too, an error is thrown.
+async function configureWeb3Account(web3) {
+    let address;
+    if (process.env.OPERATOR_ETH_PRIVATE_KEY) {
+        const account = web3.eth.accounts.privateKeyToAccount(process.env.OPERATOR_PRIVATE_KEY);
+        web3.eth.accounts.wallet.add(account);
+        address = account.address;
+    } else {
+        const accounts = await web3.eth.getAccounts();
+        if (accounts.length > 0) {
+            address = accounts[0];
+        } else {
```

Comment:
For the reviewer:
i hardcode 0 here, because everywhere in the code this same index was hardcoded. I think it's reasonable, but please comment on this.

Reply:
Yes, I think that it's reasonable. Ideally, we would create a virtual private cloud where the operator is paired with an Ethereum client and its account unlocked. I'm not sure why we passing private keys to the program instead, but I guess it works either way.

Reply:
I understand this is a point of contention between the Israeli and US teams. I honestly don't care either way, but this satisfies both use cases. For now the python infrastructure in docker-environment generates a private key and passes it in. If we change approaches in the future, no change will be needed here.